hypha/apply/activity/models.py

```diff
@@ -41,6 +41,13 @@
 
 class BaseActivityQuerySet(models.QuerySet):
     def visible_to(self, user):
+        # To hide reviews from the applicant's activity feed
+        from .messaging import ActivityAdapter
+        messages = ActivityAdapter.messages
+        if user.is_applicant:
+            return self.exclude(
+                message=messages.get(MESSAGES.NEW_REVIEW)
```

Comment:
Please add MESSAGES.NEW_REVIEW and MESSAGES.REVIEW_OPINION.

REVIEW_OPINION is a type of lightweight review.